pages/members/[slug].js

```diff
@@ -93,23 +95,33 @@ const MemberPage = ({ member, loadError }) => {
             </div>
           </div>
         }
-        <div className="columns vertical-center">
-          <div className="col">
-            <div
-              className={`profile-photo xl ${image?'has-image':'placeholder'}`}
-              style={ { backgroundImage: image && `url("${cdn}${image}-max-lg.jpg")` } }
-            >
+
+        <div className='flex flex-col md:flex-row items-start'>
+
+        <div className='flex flex-col items-start md:w-full'>
+
+            <Link  href={'/members'}>
+              <p className="text-lg cursor-pointer">
+            {'< All Profiles'}
+              </p>
+            </Link>
+            <div>
+              <img src={`${cdn}${member.photo}-profile-sm.jpg`} loading="lazy" alt="this is an image" className="w-24 mt-4 rounded-full cursor-pointer transition duration-150 transform hover:scale-110" />
```

Comment:
This needs to be conditional - if the member doesn't have a photo it should display a placeholder